### DISCUSSION TITLE
How to login different two account?

### DISCUSSION THREAD

**Discussion by ggforces** [ValvePython/steam#383] [Q&A]
import steam.webauth as wa

wa = wa.WebAuth(username="id")
wa.cli_login(password="pw")

wa = wa.WebAuth(username="id2")
wa.cli_login(password="pw2")

I got error
attributeerror: 'webauth' object has no attribute 'webauth'

**Comment by Gobot1234** [comment]
You need to avoid reassigning to the name wa try something like
```py
import steam.webauth as wa

wa_1= wa.WebAuth(username="id")
wa_1.cli_login(password="pw")

wa_2 = wa.WebAuth(username="id2")
wa_2.cli_login(password="pw2")
```

**Reply by ggforces** [reply]
How can this with while?
